stdlib/3/unicodedata.pyi

```diff
@@ -1,37 +1,36 @@
 # Stubs for unicodedata (Python 3.4)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
+from typing import Any, TypeVar, Union
 
-from typing import Any
-
-ucd_3_2_0 = ...  # type: Any
+ucd_3_2_0 = ...  # type: UCD
 ucnhash_CAPI = ...  # type: Any
 unidata_version = ...  # type: str
 
-def bidirectional(unichr): ...
-def category(unichr): ...
-def combining(unichr): ...
-def decimal(chr, default=...): ...
-def decomposition(unichr): ...
-def digit(chr, default=...): ...
-def east_asian_width(unichr): ...
-def lookup(name): ...
-def mirrored(unichr): ...
-def name(chr, default=...): ...
-def normalize(form, unistr): ...
-def numeric(chr, default=...): ...
+_default = TypeVar('_default')
+
+def bidirectional(unichr: str) -> str: ...
+def category(unichr: str) -> str: ...
+def combining(unichr: str) -> int: ...
+def decimal(chr: str, __default: _default=...) -> Union[int, _default]: ...
```

Comment:
Returning Unions is discouraged because it means you have to do an `isinstance` check whenever you use the return value. Maybe we should follow the signature of `Mapping.get`, which is now:
```
    @overload  # type: ignore
    def get(self, k: _KT) -> Optional[_VT_co]: ...
    @overload  # type: ignore
    def get(self, k: _KT, default: Union[_VT_co, _T]) -> Union[_VT_co, _T]: ...
```

Or maybe for simplicity we can just type the default as `Any` and lie slightly by saying this function always returns an int. Whatever we do, the other functions with a default argument should be the same.

Also, the chr argument to this function is also keyword-only—this appears to be true for all functions in this module.